### PR TITLE
File upload/download, link in event

### PR DESCRIPTION
**Pull request by Mathieu414**
Added a test file juste for the event "CNE", so that you can test the feature

### CODE REVIEW

**Comment by Atmos4** [comment]
Looks like you have a lot of unnecessary changes here! if you want to filter out the files you pushed by mistake, you can do `git reset --soft head~1` which will keep all your changes, but undo the last commit.

File: database/events.api.php [hunk 1]
```diff
@@ -106,4 +106,100 @@ function delete_event($event_id)
 function publish_event($event_id, $state)
 {
     return query_db("UPDATE deplacements SET open=? WHERE did=? LIMIT 1", $state, $event_id);
+}
+
+function save_registration($event_id, $user_id, $post, $competitions_id)
```
Comment (Atmos4) [inline database/events.api.php:111]: This does not belong in this PR. Please create a separate PR or remove this change.

File: pages/events/event_view.php [hunk 2]
```diff
@@ -52,15 +53,28 @@
             </div>
         </div>
 
+        <?php if ($has_file): ?>
+            <div class="file-button">
+                <a href="/download?id=<?= $event["circu"] ?>" , role="button" , class="">
```
Comment (Atmos4) [inline pages/events/event_view.php:58]: Your HTML looks weird here: you don't need commas and your class attribute is empty

Comment (Mathieu414) [reply pages/events/event_view.php]: Fixed !

**Comment by Atmos4** [comment]
database/download/Circulaire_CNE_2022.pdf This does not belong in a repository. You should add this to `.gitignore`.

File: pages/events/event_view.php [hunk 3]
```diff
@@ -52,15 +53,28 @@
...
         </div>
 
+        <?php if ($has_file): ?>
+            <div class="file-button">
+                <a href="/download?id=<?= $event["circu"] ?>" role="button">
+                    <div>
+                        <i class="fas fa-paperclip"></i>
+                    </div>
+                    <div>
+                        <b>Informations</b>
```
Comment (Atmos4) [inline pages/events/event_view.php:63]: I would have maybe displayed a bit more info here, maybe the file name? I would say that changing the structure of the table also makes sense, and maybe hashing the name of the files. However we can do that in a later PR, this is good enough for now i'd say :)

**Review by Atmos4 — APPROVED** [review]
LGTM, the few comments can be solved later!
